src/Controller/PageController.php

```diff
@@ -7,18 +7,128 @@
  * Provides route responses for the Example module.
  */
 class PageController extends ControllerBase {
```

Comment:
Should probably rename this controller to something like `CollectionsController` or something